tests/pd/mock/mocker/leader_change.rs

```diff
@@ -104,18 +104,16 @@ impl Mocker for LeaderChange {
         Some(Ok(self.resps[r.idx % self.resps.len()].clone()))
     }
 
-    fn IsBootstrapped(&self, _: &IsBootstrappedRequest) -> Option<Result<IsBootstrappedResponse>> {
-        let mut resp = IsBootstrappedResponse::new();
-        resp.set_bootstrapped(false);
-
+    fn GetRegionByID(&self, _: &GetRegionByIDRequest) -> Option<Result<GetRegionResponse>> {
```

Comment:
`GetRegionByID` always return `Err(not leader)`, it forces the `RpcClient` to update the inner leader client.